_posts/2023-05-24-cryostat-release.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Cryostat 2.3.0 is Released!
+date: 2023-05-23
+synopsis: Cryostat 2.3.0 is here. Learn what's new in this release.
+author: Andrew Azores
+toc: true
+---
+
+Hello everyone,
+
+We are pleased to announce the release of Cryostat 2.3!
+
+## What’s New in 2.3?
+- **[Cryostat Agent](/guides#using-the-cryostat-agent)**: an agent can be included in your deployment, which
+    facilitates application discovery and JFR operations over HTTP API
+- **[Cross-namespace target discovery](/get-started/#using-clustercryostats)**: a `ClusterCryostat` custom resource can
+    be configured to communicate with applications deployed across multiple namespaces
+- **Web interface enhancements**:
+  - **Quick starts and Guided Tour**: short tutorials on how to navigate and perform basic functions in Cryostat
+  - **[Topology view with bulk actions](/guides#use-topology-view)**: view your deployed applications in a topology
+  view and create JFR recordings in bulk
+  - **[Customizable dashboards](/guides#navigate-the-dashboard)**: view and configure cards displaying your JVM metrics on a dashboard
+- **Podman API**: Discover target applications using Podman HTTP API. This requires the Podman service to be running.
+    You might try `systemctl --user enable --now podman.socket`. The next time you run Cryostat under your user it will
+    be able to talk to Podman and query for running containers with the label `io.cryostat.connectUrl`. The value of
+    this label is expected to be a JMX service URL for the JVM within the container.
+
+## What’s Next?
+- Additional types of analyses and visualizations
+- Smart triggering of JFR start/stop based on JVM conditions
+
+## Where can I get the latest version of Cryostat?
+You can install Cryostat using our
+[Kubernetes operator on OperatorHub.io](https://operatorhub.io/operator/cryostat-operator)
+or via a
+[Helm Chart](https://github.com/cryostatio/cryostat-helm/tree/cryostat-v2.3)
```

Comment:
Should this link be pointing to release tag? https://github.com/cryostatio/cryostat-helm/tree/v0.3.0